come-lavoriamo/consuntivazione.md

```diff
@@ -1,62 +1,89 @@
-
-La giornata lavorativa per contratti full-time è di 8 ore. Si può decidere contrattualmente di
-ridurre le ore lavorative accedendo al cosiddetto lavoro part-time.
-
-In questa sezione si vuole offrire uno spunto per la corretta divisione del tempo lavorativo e per
-la corretta valutazione delle ore svolte sui progetti, in modo da non danneggiare né il cliente né
-Nebulab.
-
-Prima di tutto va sottolineato che lo schedule, che attulamente viene redatto sul software Float,
-rappresenta semplicemente una divisione ideale del tempo lavorativo settimanale/mensile. Ognuno di
-noi deve sapere quali sono le priorità sui progetti e deve valutare, in linea con i principi
-delineati in [Lavorare per i clienti](), di cosa si dovrà occupare in quella giornata, fermo
-restando che per la stragrande maggioranza dei casi dovrà occuparsi proprio del progetto che riporta
-lo schedule. Gli accordi con i clienti spesso prevedono un numero di ore, o più semplicemente di
-giornate, settimanali ed é importante tenerlo presente nella gestione di più progetti. Infatti
-ognuno può, se lo ritiene opportuno, anticipare una giornata su un cliente e recuperare la giornata
-dell'altro cliente in un'altro momento (comunicandolo :)).
-
-Il tema della valutazione delle ore lavorative è delicato e l'interpretazione spesso non è univoca.
-La soluzione che adottiamo a Nebulab è quella di segnare il tempo effettivamente svolto sul progetto
-avendo come unità minima la mezz'ora. Quali attività include e quali non il "tempo effettivamente
-svolto"?
-
-Include:
-
-- Tempo di implementazione effettiva
-- Tempo di pair-programming sullo stesso progetto
-- Tempo di comunicazione direttamente correlato con il lavoro che si sta svolgendo
-- Tempo di studio direttamente ed esclusivamente correlato con il lavoro che si sta svolgendo
-- Tempo di pausa normale (<30 minuti al giorno)
-
-Non include:
-
-- Tempo di implementazione su altri progetti
-- Tempo di pair-programming su altri progetti
-- Tempo di comunicazione non correlati con il lavoro che si sta svolgendo
-- Tempo di studio non direttamente e non esclusivamente correlato con il lavoro che si sta svolgendo
-- Tempo di pausa lunghi (>30 minuti al giorno)
-
-Come potete immaginare esistono diverse zone d'ombra nel confine tra attivita incluse e attività non
-incluse. La raccomandazione è quella di usare il buonsenso e di chiedere una mano ad un'altra
-persona, magari che è sul progetto con voi, se vi trovate in difficoltà nella determinazione del
-numero di ore svolte.
-
-Un piccolo trucco che potete utilizzare è cercare di quantificare le attività non incluse e
-sottrarle alle ore di presenza in ufficio.
-
-Un caso particolare: I tempi di attesa delle risposte del cliente che passo girandomi i pollici
+# Misurare il tempo
+
+In questa sezione vogliamo offrire uno spunto per la corretta determinazione da un lato delle ore
+lavorative spese sui diversi progetti e dall'altro delle ore di presenza in ufficio.
+
+### Le ore lavorative
```

Comment:
Can you use `##`? I want to maintain a logical hierarchy in the headings, so second-level headings should be under first-level headings and so on.